Makefile

```diff
@@ -9,7 +9,7 @@ build:
 	go build -v -ldflags "$(GO_LDFLAGS)" -o yace ./cmd/yace
 
 test:
-	go test -v -bench=^$ -race -count=1 ./...
+	go test -v -bench=^$$ -race -count=1 ./...
```

Comment:
@nhinds what is this needed for?

Reply:
`$` is a special character for `make`, I noticed that in the CI logs and when running `make test` locally the command was incorrect.

From ac751ac090b2d5322c0cf1c18ca1e03714833a75:

>  Escape $ in Makefile
>
> The `$` in the Makefile was being interpreted by `make`, and running the
> incorrect command `go test -v -bench=^-race -count=1 ./...`

I can send commit as its own PR if you prefer?

Reply:
Great spot! No need to open a separate PR.